src/CMakeLists.txt

```diff
@@ -22,46 +22,6 @@ set (SEQAN_APP_VERSION "${SEQAN_APP_VERSION_MAJOR}.${SEQAN_APP_VERSION_MINOR}.${
 set (MINIMUM_SEQAN_VERSION "2.3.1")
 set (MINIMUM_SEQAN3_VERSION "3.0.0")
 
-# ----------------------------------------------------------------------------
-# Dependencies (continued)
-# ----------------------------------------------------------------------------
-
-# Search SeqAn and select dependencies.
-find_package(OpenMP QUIET)
-find_package(ZLIB   QUIET)
-find_package(BZip2  QUIET)
-find_package(SeqAn  QUIET REQUIRED CONFIG)
-find_package(SeqAn3 QUIET REQUIRED CONFIG)
-
-message(STATUS "These dependencies were found:")
-message(   "     OPENMP     ${OPENMP_FOUND}      ${OpenMP_CXX_FLAGS}")
-message(   "     ZLIB       ${ZLIB_FOUND}      ${ZLIB_VERSION_STRING}")
-message(   "     BZIP2      ${BZIP2_FOUND}      ${BZIP2_VERSION_STRING}")
-message(   "     SEQAN      ${SEQAN_FOUND}      ${SEQAN_VERSION_STRING}")
-message(   "     SEQAN3     ${SEQAN3_FOUND}      ${SEQAN3_VERSION}")
-
-# Warn if Zlib was not found.
-if (NOT ZLIB_FOUND)
-  message (WARNING "WARNING: Zlib not found. Building lambda without support for gzipped input and output (this includes support for .bam).")
-endif (NOT ZLIB_FOUND)
-
-# Warn if BZip2 was not found.
-if (NOT BZIP2_FOUND)
-  message (WARNING "WARNING: BZip2 not found. Building lambda without support for bzipped input and output.")
-endif (NOT BZIP2_FOUND)
-
-if (SEQAN_VERSION_STRING VERSION_LESS "${MINIMUM_SEQAN_VERSION}")
-    message (FATAL_ERROR "The minimum SeqAn version required is ${MINIMUM_SEQAN_VERSION}!")
-    return ()
-endif ()
-
-if (SEQAN3_VERSION VERSION_LESS "${MINIMUM_SEQAN3_VERSION}")
-    message (FATAL_ERROR "The minimum SeqAn3 version required is ${MINIMUM_SEQAN3_VERSION}!")
-    return ()
-endif ()
-
-message(STATUS "The requirements were met.")
-
```

Comment:
This was just moved around, because I needed `find_package(ZLIB   QUIET)` to come after `set(CMAKE_FIND_LIBRARY_SUFFIXES ".a")` to work. I don't know why this wasn't a problem before.